package.json

```diff
@@ -14,7 +14,7 @@
 		"activeComment",
 		"chatParticipantAdditions",
 		"chatParticipantPrivate",
-		"chatSessionsProvider",
+		"chatSessionsProvider@2",
```

Comment:
The api version can only be used in extensions that have it enabled in distro, under `extensionsEnabledWithApiProposalVersion`. I assume it will just be ignored and not break anything

Reply:
Right https://github.com/microsoft/vscode/blob/422af1583191adbd102f2c3b259c73738a1ead5b/src/vs/platform/extensionManagement/common/extensionGalleryService.ts#L905-L907

Reply:
nice, thanks for checking